frontend/js/jsonify.js

```diff
@@ -0,0 +1,97 @@
+	charInfoDict["faction"] = document.getElementById("add-character-faction").value;
+	charInfoDict["race"] = document.getElementById("add-character-race").value;
+	charInfoDict["alignment"] = document.getElementById("add-character-alignment").value;
+	charInfoDict["xp"] = document.getElementById("add-character-xp").value;
+	charInfoDict["strength"] = document.getElementById("add-character-strength").value;
+	charInfoDict["dexterity"] = document.getElementById("add-character-dexterity").value;
+	charInfoDict["constitution"] = document.getElementById("add-character-constitution").value;
+	charInfoDict["intelligence"] = document.getElementById("add-character-intelligence").value;
+	charInfoDict["wisdom"] = document.getElementById("add-character-wisdom").value;
+	charInfoDict["charisma"] = document.getElementById("add-character-charisma").value;
+	charInfoDict["armourClass"] = document.getElementById("add-character-armourClass").value;
+	charInfoDict["initiative"] = document.getElementById("add-character-initiative").value;
+	charInfoDict["speed"] = document.getElementById("add-character-speed").value;
+	charInfoDict["maxHP"] = document.getElementById("add-character-maxHP").value;
+	charInfoDict["currHP"] = document.getElementById("add-character-currHP").value;
+	charInfoDict["tempHP"] = document.getElementById("add-character-tempHP").value;
+	charInfoDict["hitDice"] = document.getElementById("add-character-hitDice").value;
+	charInfoDict["deathSaves"] = document.getElementById("add-character-deathSaves").value;
+	charInfoDict["personalTraits"] = document.getElementById("add-character-personalTraits").value;
+	charInfoDict["ideals"] = document.getElementById("add-character-ideals").value;
+	charInfoDict["bonds"] = document.getElementById("add-character-bonds").value;
+	charInfoDict["flaws"] = document.getElementById("add-character-flawsname").value;
+
+	var playerInfoDict = {};
+	playerInfoDict["player"] = document.getElementById("add-character-player").value; 
+
+	var campaignInfoDict = {};
+	campaignInfoDict["campaign"] = document.getElementById("add-character-campaign").value;
+
+	//current functional code below--------------------------------------------
+
+	var jsonifiedChar = "{\"command\":\"add-character\",\"info\":{\"characterInfo\":{";
+	var player = document.getElementById("add-character-player").value;
+	var campaign = document.getElementById("add-character-name");//add campaign field to pull from? how to pull from dropdown?
+	var comma = ",";
+
+	jsonifiedChar += jsonifyField("add-character-name", "name");
+	jsonifiedChar += comma;
+	jsonifiedChar += jsonifyField("add-character-classAndLevel", "classAndLvl");
+	jsonifiedChar += "},\"playerInfo\":{\"player\":\"" + player + "\"},\"campaignInfo\":{\"campaign\":\"Campaign1\"}}}"; //need to change campaign
+	
+
+	console.log(jsonifiedChar); //just substract comma at end instead?
+
+	return jsonifiedChar;
+	//json stringify to send
+}
+
+function jsonifyField(fieldName, JSONFieldName){
```

Comment:
I don't think you're going to use this anymore, but for the sake of practice, a good name for this function could be along the lines of 'obtainValueFromField'